### PR TITLE
Xcode 10 & Swift 4.2 support

### LINKED ISSUE #68: Swift 4.2 support
RxKeybaord doesn't compile on newest Xcode 10 beta

Incompatible Swift version - framework was built with 4.1.2 (swiftlang-902.0.54 clang-902.0.39.2) and the local version is 4.2 (swiftlang-1000.0.32.1 clang-1000.10.39).

### PR DESCRIPTION
**Pull request by freak4pc**
Supersedes #72. 
Fixes #68 , #69.

### CODE REVIEW

**Comment by vaderdan** [comment]
👍 very cool!. 
Will you place binary in
https://github.com/RxSwiftCommunity/RxKeyboard/blob/master/RxKeyboard.json
just is it here the place to do it or not -> just question :)

**Comment by freak4pc** [comment]
Will merge and release later today.

File: Package.swift [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// swift-tools-version:4.1
+// swift-tools-version:4.0
```
Comment (leevlad) [inline Package.swift]: Is this not meant to be 4.2?

Comment (freak4pc) [reply Package.swift]: No - It supports both Swift 4 and Swift 4.2.

Comment (leevlad) [reply Package.swift]: Apologies. I must be doing something wrong when pulling in 0.9.0 binary via Carthage. Throws `Incompatible Swift version - framework was built with 4.0.3 ...`

Comment (freak4pc) [reply Package.swift]: Uhm that shouldn't be the case, I built it locally on my mac which has Swift 4.2. Keep trying and if you don't figure this out by weekend I'll try to help out with local testing.

**Pull request merged**